.github/workflows/tests.yaml

```diff
@@ -61,8 +61,8 @@ jobs:
 
       - name: Install stacktrace
         shell: bash -l {0}
-        # stacktrace for Python 3.8 has not been released at the moment of writing
-        if: ${{ matrix.os == 'ubuntu-latest' && matrix.python-version < '3.8' }}
+        # stacktrace for Python 3.8+ has not been released at the moment of writing
+        if: ${{ matrix.os == 'ubuntu-latest' && matrix.python-version == '3.7' }}
```

Comment:
now that 3.7 has been dropped from CI - does this mean that use of libunwind and stacktrace should be removed?

Reply:
- #5829